docs/data-guidelines.md

```diff
@@ -242,6 +242,17 @@ These conditions represent minimum requirements for the removal of valid flag da
 
 This guideline was proposed in [#6670](https://github.com/mdn/browser-compat-data/pull/6670).
 
+## When to add `version_removed` to flagged support
+
+A `version_removed` should be added to support statements containing flags under one of the following conditions, whichever comes first:
```

Comment:
```suggestion
A `version_removed` should be added to support statements containing flags under one of the following conditions:
```

Or was the "whichever comes first" significant?

Reply:
Can we express this in terms of the earliest version, rather than what comes first in the list? Then the order is no longer significant and it's easier to interpret.